.github/ISSUE_TEMPLATE/matblazor-feature-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: MatBlazor feature request
+about: Please follow this template for submitting feature requests to MatBlazor repository, thanks!
+title: ''
+labels: 'feature-request'
+assignees: ''
+
+---
+
+**Describe the feature request**
+A clear and concise description of what the feature request is.
+
+**Expected behavior**
```

Comment:
I think a better text would be in the lines of "What problem does this solve that MatBlazor isn't currently capable of" or something along those lines. What do you think?

Reply:
Yes, this sounds better, I would say. I can change this :)